virtualfish/virtual.fish

```diff
@@ -442,19 +445,44 @@ function __vf_help --description "Print VirtualFish usage information"
 end
 
 function __vf_globalpackages --description "Toggle global site packages"
```

Comment:
Due to Fish shell [variable scope](https://fishshell.com/docs/current/#variable-scope), it is probably best to explicitly declare `enabled` as a local variable here, just below the function definition:
```
set -l enabled
```
That will ensure there is no conflict with some other non-locally-scoped `enabled` variable that may have been previously defined elsewhere in the Fish environment.

Reply:
Fixed in 1395b56. Thanks for the explanations.